README.md

```diff
@@ -191,6 +191,19 @@ Code's `settings.json`:
 }
 ```
 
+## Can I use *Pyink* with the [pre-commit](https://pre-commit.com/) framework?
+
+Yes! You can put the following in your `.pre-commit-config.yaml` file:
+
+```yaml
+repos:
+  - repo: https://github.com/google/pyink
+    rev: 23.1.2
```

Comment:
These instructions won't actually work until the next release is tagged.

Reply:
Thanks, I changed this to 23.3.0 which I'll release soon after a rebase to Black's 23.3.0.